### PR TITLE
Update grid layout on the page-breadcrumb template

### PR DESCRIPTION
**Pull request by jaredcunha-usds**
## Description
This pull request is the explore how we may remove grid classes from components such as the sidenav and from the `<article>` and place them in divs that serve solely to create layout.

This also uses Formation’s [flexbox grid](https://design.va.gov/layout/flexbox-grid), along with [padding](https://design.va.gov/utilities/padding) and [margin](https://design.va.gov/utilities/margins) utilities, which gives greater control over handling responsive design than USWDS's float grid.

Perhaps at some point, we can clean these layout files to where we have them only for 1-col, 2-col, and 3-col?

## Testing done
Working on this page `http://localhost:3001/disability/how-to-file-claim/`, comparing the layout of the content that appears between the main navigation and the footer with the same page in production.

Still not sure how big of a lift this will be for the other layouts, because I didn't dig too much into how all the layouts, includes, and components are collected. 

## Screenshots
While there are only minute differences (<10px shift or so), which I think is acceptable, the main difference is at ~tablet size. I like the local version better. In production, it has always bugged me how the left navigation butts right against the side of the browser.

**Production**
Screen Shot 2019-05-24 at 11 46 48 AM

**Local**
Screen Shot 2019-05-24 at 11 47 06 AM

I took a look at this in IE11 too and seems to work

Anyway, @jbalboni and @ncksllvn, I would love to hear some of your thoughts on this. I just took a single layout to work with, but perhaps this should at least help push our conversation forward.

### CODE REVIEW

File: src/platform/site-wide/sass/shame.scss
Comment (jaredcunha-usds) [inline src/platform/site-wide/sass/shame.scss]: commented out some selectors, but this might be a little too broad. See below where I started using params on the include. That might be a safer way to handle this?

File: src/site/components/navigation-sidebar.html
Comment (jaredcunha-usds) [inline src/site/components/navigation-sidebar.html]: This is the param that allows us to override the `usa-width` property.

File: src/site/includes/detail-page.html
Comment (jaredcunha-usds) [inline src/site/includes/detail-page.html]: Here's the param in the include.

Comment (jbalboni) [reply src/site/includes/detail-page.html]: This is a temporary param so that we don't break the other layouts?

Comment (jaredcunha-usds) [reply src/site/includes/detail-page.html]: Yes… but maybe the param name should be a little broader, like `newGrid`?

File: src/platform/site-wide/sass/shame.scss
Comment (jaredcunha-usds) [inline src/platform/site-wide/sass/shame.scss]: The `new-grid` class will make the overrides easier to find, and therefore, easier to delete. That's why I'm not using utilities.

**Comment by jaredcunha-usds** [comment]
@jbalboni I'm removing the DO NOT MERGE from this. I've got the child templates from `page-breadcrumbs` situated with the new grid classes.

The pages I've been using for testing are:
**detail page:** http://localhost:3001/disability/how-to-file-claim/
**l2 index:** http://localhost:3001/disability/
**topic:** http://localhost:3001/disability/eligibility/special-claims/

I'm good with the grid. It's pretty close to what we currently have, but the rhythm is now much better than it used to be. This PR will now give us some real reusable page templates!

File: src/platform/site-wide/sass/shame.scss
Comment (jbalboni) [inline src/platform/site-wide/sass/shame.scss]: Would we be setting all articles have to 0 left and right padding once the new grid is in place across all pages? I'm not sure if we use article in places other than where you added `new-grid`, but that might be worth checking.

Comment (jaredcunha-usds) [reply src/platform/site-wide/sass/shame.scss]: Instead of setting the padding to `0`, we'd just remove the selectors altogether… both the `new-grid` and the original. That should be the same thing as setting to `0`, but I don't think it will require the rule.

**Comment by jaredcunha-usds** [comment]
@jbalboni @ncksllvn  just following up on this. was there anything you needed from me on this PR?

**Review by jbalboni — APPROVED** [review]

**Comment by jaredcunha-usds** [comment]
@jbalboni Let me know if you want me to merge this or hold until you're ready to roll out the other templates.

**Comment by jbalboni** [comment]
@jaredcunha-usds I just merged the latest from master into it, once the build finishes we can merge it.

**Pull request merged**
